src/System.Collections.Specialized/tests/HybridDictionary/GetItemObjTests.cs

```diff
@@ -14,40 +10,85 @@ public class GetItemObjTests
     {
         public const int MAX_LEN = 50;          // max length of random strings
 
+        private string[] TestValues(int length)
+        {
+            string[] values = new string[length];
+            for (int i = 0; i < length; i++)
+                values[i] = "Item" + i;
+            return values;
+        }
+
+        private string[] TestKeys(int length)
+        {
+            string[] keys = new string[length];
+            for (int i = 0; i < length; i++)
+                keys[i] = "keY" + i;
+            return keys;
+        }
+
+        private string[] Test_EdgeCases()
+        {
+            return new string[]
+            {
+                "",
+                " ",
+                "$%^#",
+                System.DateTime.Today.ToString(),
+                Int32.MaxValue.ToString()
+            };
+        }
+
+        [Theory]
+        [InlineData(50)]
+        [InlineData(100)]
+        public void Add(int size)
+        {
+            string[] values = TestValues(size);
+            string[] keys = TestKeys(size);
+
+            HybridDictionary hd = new HybridDictionary(true);
+            hd.Clear();
```

Comment:
None. Will remove. Pulled from the giant test01 where they reuse the same collection.